src/krate.rs

```diff
@@ -864,6 +894,15 @@ pub fn index(req: &mut Request) -> CargoResult<Response> {
                     .filter(crate_owners::owner_kind.eq(OwnerKind::User as i32)),
             ),
         );
+    } else if let Some(team_id) = params.get("team_id").and_then(|s| s.parse::<i32>().ok()) {
```

Comment:
For consistency with how getting a user's crates works, mostly. If we're going to pull this out into its own route, then I think users should be pulled out too, which imo should be a separate PR. WDYT?